website/static/js/logTextParser.js

```diff
@@ -495,6 +495,17 @@ var LogPieces = {
         }
     },
 
+    bitbucket_repo: {
+        view: function(ctrl, logObject) {
+            var bitbucket_user = logObject.attributes.params.bitbucket_user;
```

Comment:
Minor: as per the style guidelines, use `lowerCameCase` for variable names.

I'm aware that the github code is inconsistent--that can be changed another day...